projects/plugins/jetpack/modules/subscriptions.php

```diff
@@ -369,21 +369,6 @@ public function configure() {
 			'stc_enabled'
 		);
 
-		/** Enable Subscribe Modal */
-
-		add_settings_field(
-			'jetpack_subscriptions_comment_subscribe',
-			__( 'Enable Subscribe Modal', 'jetpack' ),
-			array( $this, 'subscribe_modal_setting' ),
-			'discussion',
-			'jetpack_subscriptions'
-		);
-
-		register_setting(
```

Comment:
Does this affect anything remaining, i.e. still possible to toggle the setting at Newsletter settings page?

Reply:
> still possible to toggle the setting at Newsletter settings page

yes, confirmed 😄